### PR TITLE
Use GraphQL for Questions tab

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/components/tickets/ticket/body/message-response.tsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { Mutation } from 'react-apollo'
 import { connect } from 'react-redux'
 import { Button } from 'semantic-ui-react'
-import { QUESTION_IS_DONE } from '../../../../features/taskmanager/queries'
+import { QUESTION_IS_DONE } from 'features/taskmanager/queries'
 import actions from '../../../../store/actions/index'
```
Comment (AlexSJ1990) [inline src/components/tickets/ticket/body/message-response.tsx:6]: this one too - but v minor

File: src/components/questions/questions-list/AnswerForm.js [hunk 2]
```diff
@@ -18,71 +27,70 @@ const FormTextArea = styled(Form.Field)`
...
+            placeholder="Answer text..."
+            onChange={answerChangeHandler}
+            value={answer}
+          />
+          <div>
+            <Button
+              style={{ marginBottom: '3px' }}
+              content="Open Chat"
+              onClick={() => chatRedirectClick()}
+            />
```
Comment (palmenhq) [inline src/components/questions/questions-list/AnswerForm.js:78]: I understand probably it was like this before but imo thi should be a `Link` not a `Button` with `redirect` :) boy scout rule and all that

File: src/features/taskmanager/queries.ts [hunk 3]
```diff
@@ -83,6 +83,6 @@ export const CHANGE_REMINDER = gql`
 
 export const QUESTION_IS_DONE = gql`
   mutation QuestionIsDone($memberId: ID!) {
```
Comment (palmenhq) [inline src/features/taskmanager/queries.ts:85]: What about renaming these to the same thing?

File: src/graphql/use-mark-question-as-done.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+import {
+  GetQuestionsGroupsDocument,
+  MarkQuestionAsDoneMutation,
+  MarkQuestionAsDoneMutationVariables,
+  useMarkQuestionAsDoneMutation,
+} from 'api/generated/graphql'
+
+export const useMarkQuestionAsDone = () => useMarkQuestionAsDoneMutation()
+
+export const markQuestionAsDoneOptions = (
```
Comment (palmenhq) [inline src/graphql/use-mark-question-as-done.tsx:11]: maybe we should prefix this with `get`? To clarify the action is not marking but rather getting the options

**Comment by vonElfvin** [comment]
Had to do this from scratch again, which is already out in production so

**Pull request closed without merging**